kaiax/gov/param.go

```diff
@@ -61,7 +60,7 @@ var (
 			// Type2. Multiple addresses, [20*n]byte.
 			// Type3. Single address, [42]byte (hex-encoded bytes). See Mainnet block 90915008.
 
-			// Type1
+			// Type1, Type3
```

Comment:
type3 is not 20 bytes
```suggestion
			// Type1
```